lib/config_builder/model/trigger.rb

```diff
@@ -0,0 +1,29 @@
+# Vagrant SSH credential model
+#
+# @see http://docs.vagrantup.com/v2/vagrantfile/ssh_settings.html
```

Comment:
Wrong content, the URL should probably point to the location of the plugin instead of the SSH documentation for vagrant.